src/functions/genfrac.js

```diff
@@ -11,24 +11,29 @@ import * as html from "../buildHTML";
 import * as mml from "../buildMathML";
 import {calculateSize} from "../units";
 
-const htmlBuilder = (group, options) => {
-    // Fractions are handled in the TeXbook on pages 444-445, rules 15(a-e).
+const adjustStyle = (size, originalStyle) => {
     // Figure out what style this fraction should be in based on the
     // function used
-    let style = options.style;
-    if (group.size === "display") {
+    let style = originalStyle;
+    if (size === "display") {
         // Get display style as a default.
         // If incoming style is sub/sup, use style.text() to get correct size.
         style = style.id > 3 ? style.text() : Style.DISPLAY;
```

Comment:
Part of previous PR, but
```suggestion
        style = style.id >= style.SCRIPT.id ? style.text() : Style.DISPLAY;
```

Reply:
Done.